src/WebSocketManager/WebSocketManagerMiddleware.cs

```diff
@@ -23,7 +23,10 @@ public class WebSocketManagerMiddleware
         public async Task Invoke(HttpContext context)
         {
             if (!context.WebSockets.IsWebSocketRequest)
+            {
+                await _next.Invoke(context);
```

Comment:
Maybe I misunderstood the error you were encountering, but I didn't just uncomment it, I also moved it to only call when the request is not a websocket request.  

It doesn't seem right to call the next middleware on every request.  Since this middleware handles (completely) the websocket requests, it shouldn't call the next middleware in the pipeline.  Only when the request is not a websocket should the next middleware execute to handle it.  If there is no other middleware, an exception appears to be the correct thing to do, since the server cannot handle the request.